src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import Footer from './index';
+
+export default {
+  title: 'Developer DAO/00-components/Footer',
```

Comment:
imo we should just flatten the titles for the time being. We can figure out organizational patterns later as we expand our component library.

```suggestion
  title: 'Footer',
```